### ISSUE TITLE
Recent activity block warnings

### ISSUE THREAD

**Issue by mudrd8mz** [academic-moodle-cooperation/moodle-mod_checkmark#15]
Two kind of warnings are displayed inside the Recent activity block:

```
Strict Standards: Only variables should be assigned by reference
in /mod/checkmark/lib.php on line 692
```

and

```
You need to update your sql to include additional name fields in the user object.

line 3616 of /lib/moodlelib.php: call to debugging()
line 2338 of /lib/weblib.php: call to fullname()
line 764 of /mod/checkmark/lib.php: call to print_recent_activity_note()
line ? of unknownfile: call to checkmark_print_recent_activity()
line 7521 of /lib/moodlelib.php: call to call_user_func_array()
line 213 of /blocks/recent_activity/block_recent_activity.php: call to component_callback()
line 74 of /blocks/recent_activity/block_recent_activity.php: call to block_recent_activity->get_modules_recent_activity()
line 288 of /blocks/moodleblock.class.php: call to block_recent_activity->get_content()
line 230 of /blocks/moodleblock.class.php: call to block_base->formatted_contents()
line 971 of /lib/blocklib.php: call to block_base->get_content_for_output()
line 1023 of /lib/blocklib.php: call to block_manager->create_block_contents()
line 392 of /lib/outputrenderers.php: call to block_manager->ensure_content_created()
line 45 of /theme/clean/layout/columns3.php: call to core_renderer->standard_head_html()
line 915 of /lib/outputrenderers.php: call to include()
line 845 of /lib/outputrenderers.php: call to core_renderer->render_page_layout()
line 245 of /course/view.php: call to core_renderer->header()
```

**Comment by kaspot** [comment]
thanks - is on our todo list

**Comment by kaspot** [comment]
Should be fixed in the newest Moodle 2.8 version.
